ios/RCTCameraManager.m

```diff
@@ -574,6 +579,11 @@ -(void)captureVideo:(NSInteger)target options:(NSDictionary *)options resolve:(R
     [self initializeCaptureSessionInput:AVMediaTypeAudio];
   }
 
+//  if ([options valueForKey:@"quality"]) {
```

Comment:
Why did you comment this out? This is where you fix it to take the value of the quality prop.